paste/fixture.py

```diff
@@ -90,13 +90,14 @@ def reset(self):
             "SMTP connection not quit")
         self.__class__.existing = None
 
+
```

Comment:
pep8 says two lines between classes

Reply:
That's what I've done here - only fixed pep8 issues with surrounding code.
Might be worth to run black once on the entire code base maybe.

Reply:
Sorry, I'm only one coffee into my morning and apparently can't read yet :)

I suspect using black would be a _huge_ change, I'm not sure we want to create that much churn, unless Paste gets truly active again.

Reply:
Agreed.
Let's postpone this.